components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -260,12 +271,12 @@ - (void)layoutSubviews {
       titleCenter.y = roundf((float)titleCenter.y);
 
     } else {
-      // Title only, center vertically.
-      // +1 for slight adjustment to font baseline for centered title labels.
-      const CGFloat centeredTitleYOffset = 1.0f;
-      titleCenter.y = contentBounds.size.height - kBottomPadding - titleSize.height +
-                      centeredTitleYOffset + ((float)titleSize.height / 2.0f);
-      titleCenter.y = roundf((float)titleCenter.y);
+      // UILabels should be aligned to pixel boundaries to avoid anti-aliasing blurriness
```

Comment:
Similar issue as the previous PR.  This will fail to pixel align on non-even font sizes.
We should round the final label.frame.origin.

Reply:
Agreed. This is the previous PR.